src/lib.rs

```diff
@@ -165,11 +165,10 @@ extern crate proc_macro;
 /// ```
 #[proc_macro_derive(ConvertGraphQLEdgesNode, attributes(graphql_client_type))]
 pub fn derive_from_graphql_client_autogen(input: TokenStream) -> TokenStream {
-    derive_from_graphql_client_autogen_2(input.clone().into())
-        .unwrap_or_else(|e| {
-            panic!("ConvertGraphQLEdgesNode macro is not correctly used for {input:?}: {e}")
-        })
-        .into()
+    match derive_from_graphql_client_autogen_2(input.clone().into()) {
+        Ok(result) => result.into(),
+        Err(e) => panic!("ConvertGraphQLEdgesNode macro is not correctly used for {input:?}: {e}"),
+    }
```

Comment:
I think `let else` would be better in this context.

Reply:
```rust
let Ok(result) = derive_from_graphql_client_autogen_2(input.clone().into()) else {
    panic!("ConvertGraphQLEdgesNode macro is not correctly used for {input:?}");
};
result.into()
```

If the code is changed to use let-else as shown above, it won’t allow access to the error in Err(e). This could result in a loss of detail in the log. Would this be acceptable?

Reply:
I am sorry. I missed that the `e` is already being handled. The original should remain unchanged.